packages/x/src/studio/models/Collection.ts

```diff
@@ -47,7 +47,7 @@ export class Collection extends BaseModel<Collection> {
    * When other collections relate to this, we need a toString() version of this collection document.
    * This can be a field reducer or an actual field, like "fullName" for `User`, or "name" for `Project`
    */
-  representedBy: Resolvable<Field>;
+  representedBy: Resolvable<Field | Field[]>;
```

Comment:
why would the fields be an array of Field?

Reply:
its array represent the path to the nested field